doorknob/tests/test_foyer.py

```diff
@@ -16,6 +20,7 @@ def test_no_emotions(self):
             with self.assertRaises(ValueError):
                 Scene(aws_data=json.load(f))
 
+
 class VideoDetectTestCase(unittest.TestCase):
     """
```

Comment:
You can delete this class declaration now, since I see that you already have the tests for ```VideoDetect``` in another file.

Reply:
I ran the SceneTestCase test class and they failed.